### DISCUSSION TITLE
Waybills are not matching and I dont know why. (BEA <-> Kristineberg)

### DISCUSSION THREAD

**Discussion by AlexEhn** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#172] [Q&A] [answered]
Based on the freight configuration I'm expecting these two freights to match but they don't. And I cannot figure out why. 

What is causing this?

![image](https://user-images.githubusercontent.com/57874863/211420970-56c81ca3-b457-4f46-ae48-bc0578f90c43.png)

**Comment by AlexEhn** [comment]
Previously I had BEA configured on package unit = Pallet, but changed it to "Not applicable" to try to get an exact match, but it does not make a difference. 

![image](https://user-images.githubusercontent.com/57874863/211421335-46b9fdba-b0e5-46bf-820f-53cb5d8f5096.png)

**Comment by fjallemark** [comment, marked as answer]
Goods without NHM-codes are only matched with the region representative station for the region your station is located in. The reason is that these goods will match almost all other stations, creating too many waybills for this type of goods. 

Only goods with NHM-codes are matched against any other module station and external stations.
